symfony/templates/participantlookup/participants-list.html.twig

```diff
@@ -30,23 +30,23 @@
             <tbody>
                 {% for participant in participants %}
                     <tr>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.lastName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.firstName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.middleName }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.dob ? participant.dob|date('n/j/Y') }}</a></td>
-                        <td><a href="{{ path('participant', { id: participant.id }) }}">{{ participant.zipCode }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.lastName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.firstName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.middleName }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.dob ? participant.dob|date('n/j/Y') }}</a></td>
+                        <td><a href="{{ path('participant', { id: participant.participantId }) }}">{{ participant.zipCode }}</a></td>
                         <td>
                             {% if participant.hpoId %}
-                                {% if participant.hpoId != app.siteOrganization %}
+                                {% if participant.awardee != siteInfo.siteAwardee %}
                                     <i class="fa fa-info-circle text-warning" title="Participant paired awardee is different from your awardee" data-toggle="tooltip"></i>
                                 {% endif %}
-                                {{ app.getAwardeeDisplayName(participant.hpoId) }}
+                                {{ siteInfo.awardeeDisplayName(participant.hpoId) }}
```

Comment:
Sorry, I missed this, I think we can use `participant.awardee` here and on line 39 as well?

Reply:
Fixed in ddb7ab6